charts/argo-cd/templates/argocd-server/deployment.yaml

```diff
@@ -41,6 +41,8 @@ spec:
         image: {{ default .Values.global.image.repository .Values.server.image.repository }}:{{ default (include "argo-cd.defaultTag" .) .Values.server.image.tag }}
         imagePullPolicy: {{ default .Values.global.image.imagePullPolicy .Values.server.image.imagePullPolicy }}
```

Comment:
Same for api server:
https://github.com/argoproj/argo-cd/blob/v2.2.5/manifests/base/server/argocd-server-deployment.yaml#L23
```yaml
        imagePullPolicy: Always
        command: [argocd-server]
        env:
```